src/components/DopplerPlus/UpgradeSuggestionForm.test.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { render, cleanup, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import UpgradeSuggestionForm from './UpgradeSuggestionForm';
+import { AppServicesProvider } from '../../services/pure-di';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { act } from 'react-dom/test-utils';
+
+describe('UpgradeSuggestionForm component', () => {
+  afterEach(cleanup);
+
+  const dependencies = {
+    appSessionRef: {
+      current: {
+        userData: {
+          user: {
+            email: 'hardcoded@email.com',
+          },
+        },
+      },
+    },
+    dopplerLegacyClient: {
+      requestSuggestionUpgradeForm: async () => {
+        return { success: true };
+      },
+    },
+    captchaUtilsService: {
+      useCaptcha: () => {
+        const Captcha = () => null;
+        const verifyCaptcha = async () => {
+          return { success: true, captchaResponseToken: 'hardcodedResponseToken' };
+        };
+        const recaptchaRef = null;
+        return [Captcha, verifyCaptcha, recaptchaRef];
+      },
+    },
+  };
+
+  const UpgradeSuggestionFormElement = () => (
+    <AppServicesProvider forcedServices={dependencies}>
+      <DopplerIntlProvider>
+        <UpgradeSuggestionForm />
+      </DopplerIntlProvider>
+    </AppServicesProvider>
+  );
+  jest.useFakeTimers();
```

Comment:
why use fake timers ? are you using setTimeout o setInterval functions ?

Reply:
@cbernat  do you know why another form tests are using fakeTimers? ( like ExclusiveForms and AgenciesForm)

Reply:
It may be an old mistake in our side 😅 We were a bit confused with timers.